tidoop-mr-lib-api/src/server.js

```diff
@@ -23,36 +23,134 @@
  * Author: frb
  */
 
-// requires
+// module dependencies
 var Hapi = require('hapi');
-var config = require('../conf/tidoop-mr-lib-api.json');
-var pjson = require('../package.json');
+var run = require('./cmd_runner.js').run;
+var version = require('../package.json').version;
+var port = require('../conf/tidoop-mr-lib-api.json').port;
+var tidoopMRLibPath = require('../conf/tidoop-mr-lib-api.json').tidoopMRLibPath;
+var tidoopMysql = require('./mysql_driver.js');
+var serverUtils = require('./server_utils.js');
 
 // create a server with a host and port
 var server = new Hapi.Server();
 
 server.connection({ 
     host: 'localhost',
-    port: config.port
+    port: port
 });
 
-// add the route
+// create a connection to the MySQL server
+tidoopMysql.connect();
+
+// add routes
 server.route({
     method: 'GET',
-    path:'/version',
+    path: '/tidoop/v1/version',
     handler: function (request, reply) {
-        console.log("Request: GET /version");
-        var response = '{version: ' + pjson.version + '}';
+        console.log("Request: GET /tidoop/v1/version");
+        var response = '{version: ' + version + '}';
         console.log("Response: " + response);
         reply(response);
-    }
+    } // handler
+});
+
+server.route({
+    method: 'GET',
+    path: '/tidoop/v1/jobs/',
+    handler: function(request, reply) {
+        console.log('Request: GET /tidoop/v1/jobs/');
+        throw new Error('Unsupported operation');
+    } // handler
+});
+
+server.route({
+    method: 'POST',
+    path: '/tidoop/v1/jobs/',
```

Comment:
Probably not important, but for clarity maybe you should use singular in this path... In case, of course only 1 job details are accepted. I mean: `path: '/tidoop/v1/job/',`